src/Service.Tests/GraphQLBuilder/Helpers/GraphQLTestHelpers.cs

```diff
@@ -43,6 +43,8 @@ public static class GraphQLTestHelpers
                     }
                     ";
 
+        public static HashSet<DatabaseType> RELATIONAL_DBS_SUPPORTING_MULTIPLE_CREATE = new() { DatabaseType.MSSQL };
```

Comment:
this hashset is now here and runtimeconfig? is the duplication necessary?

Reply:
With the removal of conditional in `MutationBuilderTests`, this hashset is not needed anymore. Removed this.